ecl/eclcc/eclcc.cpp

```diff
@@ -673,21 +674,33 @@ void EclCC::loadOptions()
     }
     extractOption(stdIncludeLibraryPath, globals, "ECLCC_ECLINCLUDE_PATH", "eclIncludePath", ".", NULL);
 
-    if (!optLogfile.length() && !optBatchMode && !optNoLogFile)
-        extractOption(optLogfile, globals, "ECLCC_LOGFILE", "logfile", "eclcc.log", NULL);
-
-    if ((logVerbose || optLogfile) && !optNoLogFile)
+    if (optLogToStdOut)
+    {
+        Owned<ILogMsgHandler> handler = getHandleLogMsgHandler(stdout);
+        handler->setMessageFields(MSGFIELD_STANDARD);
+        Owned<ILogMsgFilter> filter = getCategoryLogMsgFilter(MSGAUD_all, MSGCLS_all, optLogDetail ? optLogDetail : DefaultDetail, true);
+        queryLogMsgManager()->addMonitor(handler, filter);
+    }
+#ifndef _CONTAINERIZED
+    else
     {
-        if (optLogfile.length())
+        if (!optLogfile.length() && !optBatchMode && !optNoLogFile)
+            extractOption(optLogfile, globals, "ECLCC_LOGFILE", "logfile", "eclcc.log", NULL);
+
+        if ((logVerbose || optLogfile) && !optNoLogFile)
         {
-            StringBuffer lf;
-            openLogFile(lf, optLogfile, optLogDetail, false);
-            if (logVerbose)
-                fprintf(stdout, "Logging to '%s'\n",lf.str());
+            if (optLogfile.length())
+            {
+                StringBuffer lf;
+                openLogFile(lf, optLogfile, optLogDetail, false);
+                if (logVerbose)
+                    fprintf(stdout, "Logging to '%s'\n",lf.str());
+            }
+            if (optMonitorInterval)
```

Comment:
Should this still be happening in containerized?  Probably useful until the metrics are fully functional.

Reply:
I'm not sure useful, or will be replaced by metrics, there may well be other K8s metrics for free (dashboard) that supplant most of what it does already.
But should prob revisit all places that issue such logging separately later when metrics has progressed.